Travel/Travel/Models/Users.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Travel.Models
+{
+    public class Users
```

Comment:
This class has no identifier. Think of this class as a single table in the database.